tests/__init__.py

```
@@ -102,6 +76,10 @@
 		sys.path.insert(0, os.path.abspath(os.path.join(_BASE_NAME, _PARENT_DIR_NAME)))
 	if 'tests' in __file__:
 		sys.path.insert(0, os.path.abspath(os.path.join(_BASE_NAME, _DIR_NAME)))
+except ImportError as err:  # pragma: no branch
+	raise ImportError("[CWE-440] multicast tests Failed to import.") from err
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)